RPIPs/RPIP-44.md

```diff
@@ -22,7 +22,7 @@ There is at least one additional use case not addressed in this RPIP, which is t
 
 Execution Layer Triggerable Exits are motivated by the need to combat MEV theft by malicious Rocket Pool validators. Currently, the protocol has limited recourse against such offending validators and the proposed functionality helps to change this. It's important to continue to pursue such improvements to ensure rETH delivers the advertised APY and competes favorably with other liquid staking tokens. 
 
-A secondary motivation is to improve the node operator user experience, the ability to request exit of their validators via the protocol is an improvement to the current experience.
+A secondary motivation is to improve the node operator user experience, the ability to request exit of their validators via the protocol is an improvement to the current experience (e.g., in the case of abandonment a forced exit by the protocol would be preferable to continuous leakage penalties).
```

Comment:
Is abandonment/incapacitation examples of what this sentence was trying to hint at? I wasn't sure, and maybe including further description/example would help.

Reply:
It was more focused on the node operator side. Eg it's probably easier for a NO to just call a smart-contract function and let RP handle it than do it themselves. 

You could add the abandonment case as well, but its explicitly mentioned in the abstract as 'not included'

Reply:
More importantly than pure convenience, it means if you control the withdrawal address your principal is safe. Ie, if I lose my node address seed and light my NUC on fire, I am still ok.